ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/operators/WhereInfo.java

```diff
@@ -185,38 +187,67 @@ public boolean isWindowed() {
     return getWindowBounds().isPresent();
   }
 
-  private static List<Object> extractKeysFromInPredicate(
+  private static List<GenericKey> extractKeysFromInPredicate(
       final List<InPredicate> inPredicates,
       final LogicalSchema schema,
       final MetaStore metaStore,
       final KsqlConfig config
   ) {
+    if (schema.key().size() > 1) {
+      throw invalidWhereClauseException("Schemas with multiple "
+          + "KEY columns are not supported", false);
```

Comment:
Do we have plans to support this before 0.15? If not it might be good to slip a line calling out the limitation into the docs.

Reply:
I don't have plans at the moment, it would be relatively straightforward to add once we decide on a syntax/spec but it's not super clear to me what that spec should be. I'll document it.